focus-ios/focus-ios-tests/XCUITest/OnboardingTest.swift

```diff
@@ -4,9 +4,7 @@
 
 import XCTest
 
-class OnboardingTest: XCTestCase {
-    let app = XCUIApplication()
-
+class OnboardingTest: BaseTestCase {
```

Comment:
Let's use `BaseTestCase` so that default timeout values can be applied everywhere.